src/Microsoft.ML.Core/Environment/HostEnvironmentBase.cs

```diff
@@ -334,6 +334,10 @@ public void RemoveListener(Action<IMessageSource, TMessage> listenerFunc)
 
     public bool FallbackToCpu { get; set; }
 
+#pragma warning disable MSML_NoInstanceInitializers // Need this to have a default value.
+    protected Dictionary<string, object> Options { get; } = [];
```

Comment:
>object [](http://example.com/codeflow?start=33&length=6)

should this be `object?` to allow null settings?

Reply:
It probably isn't a bad idea. Let me make that change.

Reply:
Since we don't have nullable context enabled here skipping this for now.